shell/packages/sandstorm-ui-applist/applist.js

```diff
@@ -0,0 +1,193 @@
+SandstormAppList = function(db) {
+  this._filter = new ReactiveVar("");
+  this._sortOrder = new ReactiveVar([["appTitle", "desc"]]);
+  this._staticHost = db.makeWildcardHost("static");
+  var ref = this;
+  if (Meteor.isServer) {
+    Meteor.publish("userPackages", function() {
```

Comment:
I appear to need something substantially smarter here, because this doesn't cover all the packages that would be needed, and also this isn't reactive.  Patchset coming Soon™.